### PR TITLE
Fix incorrect condition and use of deprecated methods in PrefixRegistryImpl

### PR DESCRIPTION
**Pull request by Meeples10**
Signed-off-by: Meeples10 <8867705+Meeples10@users.noreply.github.com>

### CODE REVIEW

File: src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java [hunk 1]
```diff
@@ -28,19 +28,18 @@ public void onPrefixChange(PrefixChangeEvent event) {
...
       return;
     }
-    final MatchPlayer matchPlayer = PGM.get().getMatchManager().getPlayer(event.getUUID());
-    if (matchPlayer == null
-        || matchPlayer.getBukkit() == null
-        || matchPlayer.getBukkit().isOnline()) {
+    final Player player = Bukkit.getPlayer(event.getUUID());
+    if (player == null || !player.isOnline()) {
       return;
     }
```
Comment (Electroid) [inline src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java]: Why is the `Bukkit#getPlayer` check nessecary, can't you just use the `MatchManager#getPlayer` check?

File: src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java [hunk 2]
```diff
@@ -28,19 +28,15 @@ public void onPrefixChange(PrefixChangeEvent event) {
     if (event.getUUID() == null) {
       return;
     }
-    final MatchPlayer matchPlayer = PGM.get().getMatchManager().getPlayer(event.getUUID());
-    if (matchPlayer == null
-        || matchPlayer.getBukkit() == null
-        || matchPlayer.getBukkit().isOnline()) {
+    final Player player = Bukkit.getPlayer(event.getUUID());
+    final MatchPlayer matchPlayer = PGM.get().getMatchManager().getPlayer(player);
```
Comment (Electroid) [inline src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java]: `final MatchPlayer player = PGM.get().getMatchManager().getPlayer(event.getUUID());` ?

Comment (Meeples10) [reply src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java]: Does it matter if I use `Player#getBukkit`? I wrote it like this to avoid using that.

Comment (Electroid) [reply src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java]: What I'm saying is use what I put :)

**Pull request merged**